### PR TITLE
[t3270] Have slider container respond to clicks. Add text to report curr...

### PR DESCRIPTION
**Pull request by mjschranz**
...ent value

### CODE REVIEW

File: src/editor/sequencer-editor.js [hunk 1]
```diff
@@ -146,13 +148,25 @@ define( [ "util/mediatypes", "editor/editor", "util/time",
...
         function onSlideStart( e ) {
           e.preventDefault();
+          el.removeEventListener( "click", onClick, false );
           firstX = e.clientX;
           startLeft = scrubber.offsetLeft;
           document.addEventListener( "mousemove", onSliding, false );
           document.addEventListener( "mouseup", onSlideStop, false );
         }
 
+        function onClick( e ) {console.log("d")
```
Comment (k88hudson) [inline src/editor/sequencer-editor.js]: console.log

File: src/editor/sequencer-editor.js [hunk 2]
```diff
@@ -146,13 +148,25 @@ define( [ "util/mediatypes", "editor/editor", "util/time",
...
+          e.preventDefault();
+          var rects = el.getBoundingClientRect(),
+              percentage = ( e.clientX - rects.left ) / rects.width;
```
Comment (k88hudson) [inline src/editor/sequencer-editor.js]: Do you think this is better than mapping using a function? Is that over-engineered? If so we should probably change the other stuff that uses `.toReal` and `.toUI` to do the same thing as this

File: src/editor/sequencer-editor.js [hunk 3]
```diff
@@ -121,18 +122,19 @@ define( [ "util/mediatypes", "editor/editor", "util/time",
         }
 
         function updateUI() {
+          var sliderVal = _rootElement.querySelector( ".slider-value" );
```
Comment (k88hudson) [inline src/editor/sequencer-editor.js]: You'll want to make this a tooltip

File: src/editor/sequencer-editor.js (hunk 2 above)
Comment (mjschranz) [inline src/editor/sequencer-editor.js]: I most definitely think this is the case. It's only being used in one spot so why add function calls to the stack?

**Pull request merged**